src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -172,7 +172,11 @@ const UtilsCookie = {
 
         const applyCookieFilters = (element, filteredCookies) => {
           $(filteredCookies).each((i, cookie) => {
-            if (cookie.text !== '') {
+            if (cookie.text !== '' && $(element).is('input')) {
```

Comment:
Save $(element) in a variable in order to avoid jquery initialization multiple times

Reply:
I've change to replace jQuery to pure Javascript

Reply:
what do you mean?

Reply:
I don't see the change 😶😏.
I will review the code.

Reply:
This file is outdated, please check the last push update.
You can see the code like this to the line 172:
`        const applyCookieFilters = (element, filteredCookies) => {
          filteredCookies.forEach(cookie => {
            if (cookie.text !== '' && element.tagName === 'INPUT') {
              element.value = cookie.text
              cachedFilters[cookie.field] = cookie.text
            } else if (cookie.text !== '' && element.tagName === 'SELECT') {
              const option = document.createElement('option')
              option.value = cookie.text
              option.text = cookie.text
              element.add(option, element[1])
              element.selectedIndex = 1
              cachedFilters[cookie.field] = cookie.text
            }
          })
        }`